### DISCUSSION TITLE
Is there a blockstorage , DeviceBlockInfo, BlockRegioninfo related configuration document?

### DISCUSSION THREAD

**Discussion by snikeguo** [nanoframework/Home#1118] [General]
The configuration is too complicated...

**Comment by josesimoes** [comment]
Unfortunately there is not... we are producing documentation on a "need to have" basis. These ones are required for porting and new targets therefore have a very low usage. The relevant parts are at the top and all fields are verbosely commented so you should be able to get a good sense on what's there by looking at the existing ones.
